app/javascript/components/pages/DeckShow.js

```diff
@@ -10,22 +14,75 @@ export class DeckShow extends Component {
 		}
 	}
 
-	onClick = () => {
-		if (this.state.class === 'term') {
-			this.setState({ class: 'definition' })
+	// Toggle between "Term" and "Definition"
+	onToggle = () => {
+		if (this.state.isTerm === true) {
```

Comment:
```suggestion
  onToggle = (isTerm) => {
    this.setState({ isTerm: !isTerm});
  }
        	
```

Reply:
generally it's better to have functions to not reference `this.state` to determine the next state values; this is not a show stopper but something to keep in mind if you see that the click isn't keeping in sync with expected behavior